pcsx2/Recording/KeyMovieFrame.cpp

```diff
@@ -0,0 +1,66 @@
+#include "PrecompiledHeader.h"
+
+#include "NewRecordingFrame.h"
+
+enum {
+	File,
```

Comment:
Putting such generic names in the global scope is asking for collisions. Namespace it or name them better.